bibliopixel/drivers/WS2812SPI.py

```diff
@@ -9,7 +10,7 @@ class WS281XSPI(DriverSPIBase):
     """
 
     def __init__(self, num, **kwargs):
-        super().__init__(num, c_order=ChannelOrder.GRB, SPISpeed=3,
+        super().__init__(num, c_order=ChannelOrder.GRB, SPISpeed=3.234213120,
```

Comment:
ah forgot a comment, have also to test if `3.2` also works, `3` was to less.
don't know why but wenn I set speed throw spidev lib it is actualy this strange number.
On my first run of the SpiFileInterface the speed was false with 3Mhz so I used this strange number^^

Going to test not such a strange number.